src/sentry/api/helpers/group_index.py

```diff
@@ -493,6 +495,35 @@ def wrapper(self, request, *args, **kwargs):
     return inner
 
 
+def get_current_release_version_of_group(project_id, group_id):
+    """
+    Function that returns the latest semver release version linked to a Group or the
+    `current_release_version` for GroupResolution
+    Inputs:
+        * project_id
+        * group_id
+    Returns:
+        current_release_version
+    """
+    try:
+        # This represents the latest semver version where this group occurs in
+        order_by_semver_desc = [f"-{col}" for col in Release.SEMVER_COLS]
+        current_release_version = (
+            Release.objects.filter_to_semver()
+            .filter(
+                id__in=GroupRelease.objects.filter(
+                    project_id=project_id, group_id=group_id
+                ).values_list("release_id"),
+            )
+            .annotate_prerelease_column()
+            .order_by(*order_by_semver_desc)[:1][0]
+            .version
```

Comment:
Nit: Since you only care about `version`, you can use `.values_list("version", flat=True)` to save bringing the entire model into memory.